internal/store/backup_metadata.go

```diff
@@ -0,0 +1,289 @@
+package store
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/mattermost/mattermost-cloud/model"
+	"github.com/pkg/errors"
+)
+
+const (
+	backupMetadataTable = "BackupMetadata"
```

Comment:
I actually assumed this table name included 'Metadata' because you planned to later add one named `Backup` but if that isn't the case I agree with Gabe here

Reply:
I was thinking about it. I decided to go with `BackupMetadata` to avoid any confusion that it only stores metadata not an actual backup.
1/5 on that, I can change it if you prefer `backup`.
Btw did you only have table in mind or also the struct?

Reply:
^ Let me know what you think @gabrieljackson

Reply:
`Databases` are fair point. In API and CLI I used `backup` only so I guess we can go `Backup` to simplify.

Reply:
We chatted about this offline and all agreed that we will go with something like `InstallationBackup` to be clear in case other backup types are added in the future.